sysutils/doodle/Portfile

```
@@ -3,21 +3,27 @@
 PortSystem          1.0
 
 name                doodle
-version             0.7.0
+version             0.7.3
 description         Doodle is a tool to quickly search the documents on a computer.
 long_description    ${description}
 categories          sysutils
 license             GPL-2+
 maintainers         nomaintainer
-platforms           darwin
 
 homepage            https://grothoff.org/christian/doodle/
 master_sites        ${homepage}download/
 
-checksums           rmd160  d7b6da74efcb199fb779c023cae6f6492dd678d2 \
-                    sha256  72f84625fcf4f799ce70f0fef6364bc26433682f03c73ab64e947ec80e2edd2b
+checksums           rmd160  28fa178f45aa0653201c2359075ae58246ef4506 \
+                    sha256  aa8769d9ea58ca8960dfc30d84157efcd30b99f5e386c9f65fdb8a4d49e2bf6b \
+                    size    657044
 
-depends_lib         port:libextractor
+patchfiles-append   patchgetopt.diff
+
+configure.ldflags-append \
+                    -lintl
+
+depends_lib-append  port:libextractor \
+                    port:gettext
```

`gettext` has been split into 2 ports. There's the port containing the development headers, `gettext`, and then the runtime library is separately found as `gettext-runtime`.

```suggestion
                    port:gettext-runtime
```